### ISSUE TITLE
Calling get_class() without arguments is deprecated 

### ISSUE THREAD

**Issue by Ghost-in-Prime-69** [Automattic/WP-Job-Manager#2843]
It's not actually a security issue, but you should still know about this. In PHP 8.3 wp-job-manager/includes/class-wp-job-manager-usage-tracking.php on line 163 the function get_class() is empty and therefore deprecated.

**Comment by masteradhoc** [comment]
This seems to be fixed by #2836. Do you still see this issue @Ghost-in-Prime-69?

**Comment by Ghost-in-Prime-69** [comment]
Yes you are right it is fixed by now. Gonna mark this as solved!
